src/reduxStore.jsx

```diff
@@ -1,6 +1,6 @@
 import { createStore, applyMiddleware, combineReducers } from 'redux';
 import thunk from 'redux-thunk';
-import reducer from './reducer';
+import transactions from './Stats/reducer';
```

Comment:
Transactions sounds better, we should call the folder Transactions instead of Stats

Reply:
fixed